what-is-chalk.mdx

```diff
@@ -18,215 +18,440 @@ import { PyDiffEditor } from '@/components/Editor'
+
+**No DSLs, building with Chalk is building with Python**
 
-Once you promote your deployment to production, Chalk makes features
-available for low-latency [online inference](/docs/query-basics) and
-[offline training](/docs/query-offline). Chalk uses
-the **exact same source code** to serve [temporally-consistent](/docs/temporal-consistency) training
-sets to data scientists and live feature values to models. This re-use
-dramatically shortens development time and ensures that feature values
-from online and offline contexts match.
+Chalk gives MLOps teams the tools to deploy enterprise-grade infrastructure with a superb developer experience. Define features in Python that are run natively in Rust, C++, or as UDFs—no need to manually port notebooks and code into other languages e.g. Scala.
 
-### Online inference
+Chalk takes it a step further by democratizing modern software engineering practices for data teams:
 
-Chalk's online store & feature computation engine make it easy to query
-features with ultra low-latency for your online inference use cases.
+- features are discoverable and auditable across all environments
+- experimental changes are isolated to branches and safely promoted via CLI with gradual rollouts
+- models are versioned and can be easily rolled-back the same way you would an API
 
-Integrating Chalk with your production application takes minutes via
-Chalk's simple REST API:
+With SDKs available in multiple languages, you can easily serve features across your entire tech stack, empowering everyone from data scientists using Python to frontend developers in JavaScript and backend engineers working with Java.
 
 <RequestingFeaturesOnline style={{ height: 328 }} />
 
-Features computed to serve online requests are also replicated to Chalk's
-offline store for historical performance tracking and training set generation.
+**Observability**
 
-### Offline training
+Chalk enables teams to monitor and trace features back to the source, reducing governance bottlenecks by providing audit-ready records of all inputs and outputs for compliance.
 
-Data scientists can use Chalk's Jupyter integration to create datasets
-and train models. Datasets are stored and tracked so that they can be
-re-used by other modelers. Chalk implements model provenance to track
-inputs, outputs, and other data for audit and reproducibility.
+- tracks the inputs, outputs and latencies of every step in any inference for debugging and monitoring
+- provides detailed feature lineage showing data provenance and transformation history
+- enables real-time monitoring of feature drift, access patterns, and performance across model deployments
 
-<JupyterNotebook />
+Chalk transforms observability from a technical challenge into a strategic advantage, giving you the confidence to deploy and iterate on ML systems with full visibility and control.
```

Comment:
technical challenge -> strategic advantage is a little odd as a transition. i think some more parallel would sound nicer here (vulnerability -> strength)